widget/IncompatiblePointDistribution.tsx

```diff
@@ -66,7 +75,9 @@ class IncompatiblePointDistribution extends Component<IncompatiblePointDistribut
     if (this.props.selectedDataPoint != null) {
       // Sort the data into the dataRows based on the ordering of the sorted classes
       var totalErrors = 0;
-      for (var i=0; i < this.props.selectedDataPoint.sorted_classes.length; i++) {
+      var start_i = this.state.page * this.props.pageSize;
```

Comment:
Minor nitpick here. If we define and use variables within the Typescript we try and use camel case if possible, i.e. `startI` instead of `start_i`.

The exception to this would be if we are dealing with objects derived from the Python, where we use so called snake case naming for our variables.